src/test/java/uk/gov/hmcts/reform/civil/service/robotics/mapper/RoboticsDataMapperTest.java

```diff
@@ -21,6 +21,7 @@
 import uk.gov.hmcts.reform.civil.model.SolicitorOrganisationDetails;
 import uk.gov.hmcts.reform.civil.model.referencedata.response.LocationRefData;
 import uk.gov.hmcts.reform.civil.model.robotics.NoticeOfChange;
+import uk.gov.hmcts.reform.civil.model.referencedata.response.LocationRefData;
```

Comment:
this feels like duplicate import here

Reply:
You are better than checkstyle ;) I fix it.